app/src/main/java/com/github/fribourgsdp/radio/MusixmatchLyricsGetter.kt

```diff
@@ -32,147 +30,173 @@ interface LyricsGetter{
  * API Call doc : https://stackoverflow.com/questions/45219379/how-to-make-an-api-request-in-kotlin
  */
 object MusixmatchLyricsGetter : LyricsGetter {
-        const val LYRICS_NOT_FOUND = "---No lyrics were found for this song.---"
+    const val LYRICS_NOT_FOUND = "---No lyrics were found for this song. Feel free to add yours !---"
```

Comment:
Why do you save the string in a constant instead of directly using R.string.(...)?

Reply:
Because this object has no context. But this is not a problem since it is never displayed as-is but in SongFragment it takes the value from the strings.xml file. For clarity I shall rename it.